### ISSUE TITLE
NPcap: C:\npcap-sdk-0.1\Include

### ISSUE THREAD

**Issue by david-delaune** [nmap/nmap#1875]
Yeah,

Your team has hard-coded the npcap include paths. Please change them to use a relative path.

**Comment by dmiller-nmap** [comment]
I agree this isn't the cleanest way. What I want is a single place where the path to the Npcap SDK can be configured, so that a user can easily copy one of the Examples to make a new project. I thought I might define a parameter on the msbuild command line and then reference that in the project files, but I couldn't figure out how to do that back when I updated the Examples directory. Do you know a good way to do that?

**Comment by david-delaune** [comment]
> Do you know a good way to do that?

Yes, by the way the GNU makefiles look fine. However the Microsoft Visual Studio projects needs to be modified to use relative paths. You can take advantage of the [common macros](https://docs.microsoft.com/en-us/cpp/build/reference/common-macros-for-build-commands-and-properties?view=vs-2019) 

Instead of using hard coded paths you can use **$(SolutionDir)\..\Include**
